src/sas/system/style.py

```diff
@@ -0,0 +1,73 @@
+import os
+import pkg_resources
+import sys
+
+from sas.system import user
+
+DEFAULT_STYLE_SHEET_NAME = 'sasview.css'
+
+
+class StyleSheet:
+    """
+    The directory where the per-user style sheet is stored.
+
+    Returns ~/.sasview/sasview.css, creating it if it does not already exist.
+    """
+    def __init__(self):
+        self.style_sheet = None
+        self._find_style_sheet()
+        self.save()
+
+    def style_sheet_filename(self):
+        """Filename for saving config items"""
+        user_dir = user.get_user_dir()
+        self.style_sheet = os.path.join(user_dir, DEFAULT_STYLE_SHEET_NAME)
+
+    def save(self):
+        self._find_style_sheet()
+        sheet = self()
+        self.style_sheet_filename()
+        with open(self.style_sheet, 'a') as file:
+            file.write(sheet)
+
+    def _find_style_sheet(self, filename=DEFAULT_STYLE_SHEET_NAME):
```

Comment:
if this does come from config, this would have to be replaced with an optional because it keyword defaults are evaluated at compile time